docs/about/meetings.rst

```diff
@@ -5,7 +5,25 @@ The CIViC team regularly holds curation jamborees and development hackathons. At
 Upcoming Meetings & Events
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-:guilabel:`No upcoming events currently scheduled.`
+:guilabel:`Next planned event.`
+
+.. rubric:: 2022 CIViC Hackathon and Curation Jamboree
+
+----
+
+:Dates: Jul 30th, 2022
+:Location: Washington University in St. Louis
+:Sign Up: `Eventbrite: Cancer Variant Interpretation Hackathon and Curation Jamboree 2022 <https://eventbrite.com/e/cancer-variant-interpretation-hackathon-and-curation-jamboree-tickets-48287431006>`_
```

Comment:
This is the wrong link.

Reply:
Should be this one: https://www.eventbrite.com/e/cancer-variant-interpretation-hackathon-and-curation-jamboree-tickets-327320021997

Reply:
Good catch @kkrysiak !